idflakies-maven-plugin/src/main/java/edu/illinois/cs/dt/tools/plugin/DetectorMojo.java

```diff
@@ -350,4 +371,24 @@ private static List<Runner> removeZombieRunners(
         }
         return aliveRunners;
     }
+
+    public static void timing(long startTime) {
+        if (!Files.exists(PathManager.timePath())) {
+            try {
+                Files.createFile(PathManager.timePath());
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+        long endTime = System.currentTimeMillis();
+        double duration = (endTime - startTime) / 1000.0;
+
+        String time = duration + ",";
+        try {
+            Files.write(PathManager.timePath(), time.getBytes(),
```

Comment:
It seems to be writing out the time appended always to the same file, so can get confusing which time goes with whatever run...